### PR TITLE
Build docker image in build workflow; Update automatic install to use Docker

### PR DESCRIPTION
**Pull request by jamilbk**
Builds the prod Docker image and pushes to `ghcr.io/firezone/firezone` for internal testing / review.

The `publish` step will build and push the prod image to DockerHub.

This also upgrades OTP to 25.1 to fix a [critical severity CVE](https://github.com/advisories/GHSA-f399-ff4w-62fm) in OTP's SSL implementation. Note Firezone is unaffected since we don't terminate SSL in OTP.

### CODE REVIEW

File: .github/workflows/docker_build.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      IMAGE_NAME: ${{ github.repository }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/amd64/v2
+          - linux/amd64/v3
+          # TODO: Build for other platforms. Will likely need our own
+          # runners for each of these since QEMU segfaults for most of
```
Comment (conectado) [inline .github/workflows/docker_build.yml]: Weird, do you know why it segfaults?

**Review by conectado — DISMISSED** [review]
Looks good, just left a comment because I don't know why QEMU segfaults.

File: .github/workflows/docker_build.yml (hunk 1 above)
Comment (jamilbk) [reply .github/workflows/docker_build.yml]: Yeah it happens due to the OTP 25 JIT.

Hmm. I can build it without the JIT, and we can probably have cross platform then. Let me give that a shot.

Comment (jamilbk) [reply .github/workflows/docker_build.yml]: https://gitlab.com/qemu-project/qemu/-/issues/1034

Comment (jamilbk) [reply .github/workflows/docker_build.yml]: Building a new base image here:

https://github.com/firezone/containers/actions/runs/3131631443

Comment (conectado) [reply .github/workflows/docker_build.yml]: Seems like it failed due to broken github actions

Comment (conectado) [reply .github/workflows/docker_build.yml]: But with latest main it did work, so yeah.

I wonder if there's a significant loss of performance by disabling JIT?

File: scripts/install.sh [hunk 2]
```diff
@@ -78,87 +100,93 @@ kernelCheck() {
...
+  cd -
 }
 
-firezoneSetup() {
-  conf="/opt/firezone/embedded/cookbooks/firezone/attributes/default.rb"
-  sed -i "s/firezone@localhost/$1/" $conf
-  sed -i "s/default\['firezone']\['external_url'].*/default['firezone']['external_url'] = 'https:\/\/$public_ip'/" $conf
-  firezone-ctl reconfigure
-  firezone-ctl create-or-reset-admin
+displayLogo() {
```
Comment (conectado) [inline scripts/install.sh:129]: :rocket:

**Review by conectado — APPROVED** [review]
Awesome!

**Pull request merged**